### PR TITLE
Add new non-mediawiki homepage

### PR DESCRIPTION
**Pull request by rizend**
Adds new homepage and makes changes to caddy config so that `/` is routed to the homepage instead of mediawiki.

### CODE REVIEW

File: site.yml [hunk 1]
```diff
@@ -20,6 +20,14 @@
     - mediawiki
     - cloudalchemy.node-exporter
     - cloudalchemy.prometheus
+  tasks:
```
Comment (SuperQ) [inline site.yml]: Let's keep this kind of thing out of the site playbook. Either stash this change in the `mediawiki` role, or create a new `home_page` role with a new tasks/main.yml.

Comment (rizend) [reply site.yml]: I've stashed it in the `mediawiki` role.

File: group_vars/noisebridge-net/caddy.yml [hunk 2]
```diff
@@ -43,7 +43,7 @@ caddy_config: |
 
     root /srv/mediawiki/noisebridge.net
 
-    index index.php
+    index home_page.html index.php
```
Comment (SuperQ) [inline group_vars/noisebridge-net/caddy.yml]: Should we just follow the common `index.html` convention?

Comment (rizend) [reply group_vars/noisebridge-net/caddy.yml]: Yeah, I'd actually prefer that.  I was originally thinking that I didn't want to overwrite an `index.html` already present which is why I chose the non-default name in the first place.

Comment (SuperQ) [reply group_vars/noisebridge-net/caddy.yml]: `index.html` is not present, only `index.php`. So a setting of `index index.html index.php` should work fine.

Comment (rizend) [reply group_vars/noisebridge-net/caddy.yml]: Latest commit makes the switch

**Comment by SuperQ** [comment]
Nice!

**Pull request merged**